tools/src/code-review/reviewers/checkMissingChangelogs.ts

```diff
@@ -67,17 +69,32 @@ function relativeChangelogPath(head: ReviewInput['pullRequest']['head'], pkg: Pa
   return `[\`${relativePath}\`](${head.repo?.html_url}/blob/${head.ref}/${relativePath})`;
 }
 
-function missingChangelogOutput(changelogLinks: string): ReviewOutput {
+function missingChangelogOutput(changelogLinks: string, pullRequest: PullRequest): ReviewOutput {
   return {
     status: ReviewStatus.WARN,
     title: 'Missing changelog entries',
     body:
-      'Your changes should be noted in the changelog. ' +
+      'Your changes should be noted in the changelog, e.g.: \n' +
+      generateChangelogStub(pullRequest) +
+      '\n' +
       'Read [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md) ' +
       `guide and consider adding an appropriate entry to the following changelogs: \n${changelogLinks}`,
   };
 }
 
+function generateChangelogStub(pullRequest: PullRequest) {
+  const prLink = markdownLink('#' + pullRequest.number, pullRequest.html_url);
+  const userLink = markdownLink('@' + pullRequest.user!.login, pullRequest.user!.html_url);
+  const suggestedTitle = filterBracketContent(pullRequest.title);
+
+  return `- ${suggestedTitle} (${prLink} by ${userLink})`;
+}
+
+function filterBracketContent(input: string): string {
+  // many PR titles start with package name (e.g. [video]). We don't want that in a package-specific changelog.
+  return input.replace(/\[.*?\]/g, '').trim();
```

Comment:
It's not always true actually. We usually do want to keep platform-specific tags such as `[Android]` or `[iOS]`. Maybe it doesn't matter here though. I think people in the team mostly just add the entry to the changelog locally without links and apply bot's suggestions before merging. In many cases, the changelog entry is also different than PR's title.

Reply:
Hi, true, I forgot about those. It's a suggestion so the assumption is that they'll be tweaked.